enhancements/node-tuning/pao-in-nto.md

```diff
@@ -0,0 +1,317 @@
+---
+title: pao-as-part-of-core-ocp
+authors:
+  - "@MarSik"
+  - @yanirq
+reviewers:
+  - @fromanirh
+  - @jmencak
+  - @mrunalp
+  - @dhellmann
+approvers:
+  - TBD
+creation-date: 2021-08-12
+last-updated: 2021-08-12
+status: provisional
+---
+
+# PAO as part of core OCP
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [x] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
```

Comment:
This section is really intended to be a summary of the proposal, not so much a summary of the state of existing code. I would keep the first paragraph, and add a line or two about wanting to add all of the existing implementation of PAO to NTO without adding new features. Then I would move most of the rest of the content down to the `Proposal` or `Implementation Details` section. If you put it in the `Proposal` section, you could add a new subsection there with a name like `Existing Implementation`.